src/components/Tool/Import/Csv/CsvValues.test.tsx

```diff
@@ -0,0 +1,357 @@
+import React from 'react';
+import { ThemeProvider } from '@emotion/react';
+import { render, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TestWrapper from '__tests__/util/TestWrapper';
+import { useApiConstants } from 'src/components/Constants/UseApiConstants';
+import theme from 'src/theme';
+import {
+  CsvImportContext,
+  CsvImportType,
+  CsvImportViewStepEnum,
+} from './CsvImportContext';
+import CsvValues, { CsvValuesProps } from './CsvValues';
+import { save } from './csvImportService';
+
+jest.mock('./csvImportService');
+jest.mock('src/components/Constants/UseApiConstants');
+
+let uploadData;
+const setUploadData = jest.fn();
+
+let initialData;
+const setInitialData = jest.fn();
+
+const setCurrentTab = jest.fn();
+
+const constants = {
+  sendAppeals: [
+    { id: 'true', value: 'Yes' },
+    { id: 'false', value: 'No' },
+  ],
+  newsletter: [
+    {
+      id: 'Physical',
+      value: 'Physical',
+    },
+    {
+      id: 'Email',
+      value: 'Email',
+    },
+    {
+      id: 'Both',
+      value: 'Both',
+    },
+    {
+      id: 'None',
+      value: 'None',
+    },
+  ],
+  pledgeCurrency: [
+    {
+      code: 'USD',
+      codeSymbolString: 'USD $',
+      name: 'USD',
+    },
+  ],
+};
```

Comment:
I feel like you have a lot of mock data which is the same across muliple files, you could move the data into a mock file.